DataProcessing/QuiverWallStreetBetsDataDownloader.cs

```diff
@@ -293,96 +204,39 @@ private async Task<string> HttpRequester(string url)
                 }
             }
 
-            throw new Exception($"Request failed with no more retries remaining (retry {_maxRetries}/{_maxRetries})");
+            throw new Exception($"QuiverWallStreetBetsDataDownloader.HttpRequester(): Request failed with no more retries remaining (retry {_maxRetries}/{_maxRetries})");
         }
 
         /// <summary>
-        /// Saves contents to disk, deleting existing zip files
+        /// Saves contents to disk
         /// </summary>
         /// <param name="destinationFolder">Final destination of the data</param>
-        /// <param name="name">file name</param>
+        /// <param name="name">File name</param>
         /// <param name="contents">Contents to write</param>
         private void SaveContentToFile(string destinationFolder, string name, IEnumerable<string> contents)
         {
-            name = name.ToLowerInvariant();
-            var finalPath = Path.Combine(destinationFolder, $"{name}.csv");
-            var finalFileExists = File.Exists(finalPath);
+            var lines = new HashSet<string>(contents).ToList();
 
-            var lines = new HashSet<string>(contents);
-            if (finalFileExists)
+            if (!destinationFolder.Contains("universe"))
             {
-                foreach (var line in File.ReadAllLines(finalPath))
-                {
-                    lines.Add(line);
-                }
+                lines = lines
+                    .OrderBy(x => DateTime.ParseExact(x.Split(',').First(), "yyyyMMdd", CultureInfo.InvariantCulture,
+                        DateTimeStyles.AdjustToUniversal))
+                    .ToList();
```

Comment:
Previous implementation was merging with any existing data on disk, current isn't? I think previous would also order universe files which seems like a nice to have behavior?

Reply:
I see your point: if Quiver database removes the old data, we would still save have it. I will also order the universe by SID.